apps/backoffice-v2/src/pages/Entities/hooks/useEntities/useEntities.tsx

```diff
@@ -20,11 +21,8 @@ export const useEntities = () => {
     pageSize,
     search,
   });
-
-  const {
-    meta: { totalPages },
-    data: workflows,
-  } = data || { meta: { totalPages: 0 }, data: [] };
+  const cases = data?.data;
+  const totalPages = data?.meta?.totalPages ?? 0;
```

Comment:
Do you think we should use `useMemo` here?

Reply:
No. There is no computation here, no multiple references made (i.e. an object inside an array), and these values are not used in many `useEffect` calls or similar.